test/tests/functional/pbs_acl_host_moms.py

```diff
@@ -52,40 +52,40 @@ def setUp(self):
 
         TestFunctional.setUp(self)
 
-        if len(self.moms) != 2:
-            self.skip_test('test requires two MoMs as input, ' +
-                           '  use -p moms=<mom1>:<mom2>')
+        if len(self.moms) != 1:
+            self.skip_test('test requires one MoM as input, ' +
+                           '  use -p moms=<mom>')
```

Comment:
Have deleted the hook after the test case. Also removed requirement for 2 moms since only one is required.